### PR TITLE
documentation my favorite! and a chess board!

### PR DESCRIPTION
**Pull request by jrutishauser**

### CODE REVIEW

**Comment by carolinapowers** [comment]
:+1:

**Comment by sak55** [comment]
:+1:

**Comment by jrutishauser** [comment]
https://github.com/TheIronYard--Orlando/FEE--2015--SPRING/issues/121

**Comment by jgarcia99** [comment]
:+1:

File: index.html [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+<div class="square"><span>8</span></div>
+<div class="square"><span></span></div>
+<div class="square"><span></span></div>
+<div class="square"><span></span></div>
+<div class="square"><span></span></div>
+<div class="square"><span></span></div>
+<div class="square"><span></span></div>
+<div class="square"><span></span></div>
+</div>
+</div>
```
Comment (al-the-x) [inline index.html:104]: Nice indentation, bro. Highly readable. :trollface:

File: index.html [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+        <script>
+            (function(b,o,i,l,e,r){b.GoogleAnalyticsObject=l;b[l]||(b[l]=
+            function(){(b[l].q=b[l].q||[]).push(arguments)});b[l].l=+new Date;
+            e=o.createElement(i);r=o.getElementsByTagName(i)[0];
+            e.src='//www.google-analytics.com/analytics.js';
+            r.parentNode.insertBefore(e,r)}(window,document,'script','ga'));
+            ga('create','UA-XXXXX-X');ga('send','pageview');
+        </script>
+    </body>
+</html>
```
Comment (al-the-x) [inline index.html:124]: So, I think this was supposed to replace `pub/index.html`... but you probably already know that by now.

**Pull request merged**